data.yaml

```diff
@@ -1,11 +1,11 @@
 active_epochs: ["2023.7"]
 next_epoch: "2023.9"
-distros: [core, community]
+distros: [core, community, minimal]
 packages:
 
 - name: provenance-lib
   repo: qiime2/provenance-lib
-  distros: [core, community]
+  distros: [core, community, minimal]
```

Comment:
Does provenance replay work on plugins that aren't installed?

Asking because it might limit how useful having this installed is if it can't be used on most qzas.

Reply:
It doesn't, but would be useful for replay of anything generated with plugins installed in this distro. I was mainly including it for consistency, since it'll be part of the framework soon. I feel like it might as well be included, but I'm ok not including it if you think it shouldn't be there.